src/main/scala/ch/epfl/bluebrain/nexus/sbt/nexus/ServicePackagingPlugin.scala

```diff
@@ -0,0 +1,72 @@
+package ch.epfl.bluebrain.nexus.sbt.nexus
+
+import java.util.regex.Pattern
+
+import com.typesafe.sbt.packager.Keys._
+import com.typesafe.sbt.packager.archetypes.JavaAppPackaging
+import com.typesafe.sbt.packager.docker.DockerPlugin.autoImport.{Docker, DockerAlias}
+import com.typesafe.sbt.packager.docker.{Cmd, DockerPlugin, ExecCmd}
+import com.typesafe.sbt.packager.universal.UniversalPlugin
+import sbt.Keys._
+import sbt._
+
+/**
+  * Plugin defining default configuration for building docker images for akka based service implementations.
+  */
+object ServicePackagingPlugin extends AutoPlugin {
+
+  override lazy val requires = UniversalPlugin && JavaAppPackaging && DockerPlugin
+
+  override lazy val trigger = noTrigger
+
+  override lazy val projectSettings = Seq(
+    maintainer         := "Nexus Team <noreply@epfl.ch>",
+    dockerBaseImage    := "openjdk:8-jre",
+    daemonUser         := "root",
+    dockerExposedPorts := Seq(8080, 2552),
+    dockerRepository   := sys.env.get("DOCKER_REGISTRY"),
+    dockerAlias := DockerAlias(dockerRepository.value,
+                               None,
+                               (packageName in Docker).value,
+                               Some((version in Docker).value)),
+    dockerBuildOptions ++= {
+      val options = for {
+        stringArgs <- sys.env.get("DOCKER_BUILD_ARGS").toList
+        arg        <- stringArgs.split(Pattern.quote("|"))
+        pair       <- List("--build-arg", arg)
+      } yield pair
+      options
+    },
+    dockerUpdateLatest          := !isSnapshot.value,
```

Comment:
I think we should not update latest from SBT, as it will generate a new docker image to push. We can do that as part of the build.